src/Goteo/Model/Faq/FaqSection.php

```diff
@@ -111,63 +98,82 @@ static public function getList(array $filters = [], int $offset = 0, int $limit
               $sql
               ORDER BY faq_section.order ASC
               LIMIT $offset, $limit";
-         //die(\sqldbg($sql, $values));
         $query = static::query($sql, $values);
         return $query->fetchAll(\PDO::FETCH_CLASS, __CLASS__);
     }
 
+     static public function getListCount(array $filters = [], $lang = null): int
+     {
+         if(!$lang) $lang = Lang::current();
+         list($fields, $joins) = self::getLangsSQLJoins($lang, Config::get('sql_lang'));
+
+         $filter = [];
+         $values = [];
+
+         if($filter) {
+             $sql = " WHERE " . implode(' AND ', $filter);
+         }
```

Comment:
The variable `$filter` will always be empty (since it has just been defined as empty 2 lines above), so this IF clause is never going to happen.

The same happens in the `getList()` method. The `$filter` variable is always an empty array.

Either we should be checking for `$filters` (the method parameter), or this IF could just be removed.

I think that issue is that we should be checking if `$filters` has any item inside instead of `$filter`.